src/Tasks/Common/Resources/Strings.resx

```diff
@@ -484,7 +484,7 @@ The following are names of parameters or literal values and should not be transl
     <comment>{StrBegin="NETSDK1094: "}</comment>
   </data>
   <data name="ReadyToRunTargetNotSuppotedError" xml:space="preserve">
```

Comment:
Can we fix the typo (`Suppoted` -> `Supported`) in this resource name?

Reply:
Fixed.